src/php/Command/CommandConfig.php

```diff
@@ -32,7 +32,7 @@ public function getReplStartupFile(): string
 
     public function getApplicationRootDir(): string
     {
-        return Config::getInstance()->getApplicationRootDir();
+        return Config::getInstance()->getAppRootDir();
     }
```

Comment:
You can remove this `getApplicationRootDir()` completely from the Configs classes, and use the `getAppRootDir()` directly from the Factories.